ingwarsw/tinyfugue#26 introduced a regression where the flag to set the
background color overlapped with the bottom of the foreground color.
In the case where the foreground color was odd, it would set the background
color.
This should fix the regression, albeit in a rather ugly way.
diff --git a/src/tf.h b/src/tf.h
index d5bd672227da..685d3d6b28de 100644
--- a/src/tf.h
+++ b/src/tf.h
@@ -79,18 +79,18 @@ enum enum_attr {
     F_FLASH       = 0x0000,   /* zero - not implemented */
     F_DIM         = 0x0000,   /* zero - not implemented */
     F_BOLD        = 0x0004,
-    F_ITALIC      = 0x0008,
-    F_HILITE      = 0x0010,
-    F_NONE        = 0x0020,
-    F_EXCLUSIVE   = 0x0040,
+    F_ITALIC      = 0x80000000, /* this isn't pretty, but it fixes a regression */
+    F_HILITE      = 0x0008,
+    F_NONE        = 0x0010,
+    F_EXCLUSIVE   = 0x0020,
 
 #if NCOLORS == 256 /* XXX ??? */
 # define FGCOLORSHIFT 8
     F_FGCOLORMASK = 0x0000ff00,   /* 8 bits, interpreted as an integer */
-    F_FGCOLOR     = 0x00000080,   /* flag */
+    F_FGCOLOR     = 0x00000040,   /* flag */
 # define BGCOLORSHIFT 16
     F_BGCOLORMASK = 0x00ff0000,   /* 8 bits, interpreted as an integer */
-    F_BGCOLOR     = 0x00000100,   /* flag */
+    F_BGCOLOR     = 0x00000080,   /* flag */
 #else
     /* inside the 16 low bits */
 # define FGCOLORSHIFT 8
